sass/themes/_error.scss

```
@@ -1,5 +1,39 @@
 @charset 'UTF-8';
 
+/*doc
+---
+title: "Attribute should not contain whitespace"
+name: attribute-whitespace
+category: Errors
+---
+
+## Description
+Some HTML attributes should not contain any whitespace, as any `[id]` or `[lang]` attribute, or `map[name]`.
+
+## Reference
+* <https://html.spec.whatwg.org/#the-id-attribute>
+* <https://html.spec.whatwg.org/#the-map-element>
+* <https://tools.ietf.org/html/bcp47#section-2.1.1>
+
+## Selector
+```css_example
+[id*=" "],
+[lang*=" "],
+map[name*=" "]
+```
+
+## Test
+```html_example
+<p id="my id">This is my beautiful element with an ID</p>
+```
+*/
+[id*=" "],
+[lang*=" "],
+map[name*=" "] {
+  @include error('attribute-whitespace');
```

Since it can target void tags too, we should extend this:
```suggestion
  @include error('attribute-whitespace');
  
  @include void-tags {
    @include error('attribute-whitespace', $self-closing: true);
  }
```